src/source/YamlGenerator.scala

```diff
@@ -103,11 +103,12 @@ class YamlGenerator(spec: Spec) extends Generator(spec) {
         }.mkString(" deriving(", ", ", ")")
       }
     }
-    td.body match {
-      case i: Interface => "interface" + ext(i.ext)
-      case r: Record => "record" + ext(r.ext) + deriving(r)
-      case e: Enum => "enum"
-    }
+  	td.body match {
```

Comment:
I'm seeing inconsistent indentation in this block.  Are you mixing tabs and spaces?  Spaces are preferred.